atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceTableCreationIntegrationTest.java

```diff
@@ -58,7 +60,7 @@ public void setUp() {
                 CassandraKeyValueServiceConfigManager.createSimpleManager(quickTimeoutConfig), CassandraTestSuite.LEADER_CONFIG);
 
         ImmutableCassandraKeyValueServiceConfig slowTimeoutConfig = CassandraTestSuite.CASSANDRA_KVS_CONFIG
-                .withSchemaMutationTimeoutMillis(60 * 1000);
+                .withSchemaMutationTimeoutMillis(6 * 1000);
```

Comment:
Yes - so, originally, the test didn't have an `awaitTermination` check, so:
a) it didn't actually test what was intended
b) the threads continued after the test was done, causing tables to bleed into other tests

I think that bumping this timeout down was necessary to get this test running in a reasonable amount of time, since other threads seemed to be waiting for this long before trying again to get the schemaMutationLock.